packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -169,8 +137,8 @@ export class EditorStore implements CommandRegistrar {
   sdlcState: EditorSDLCState;
   graphState: EditorGraphState;
   graphManagerState: GraphManagerState;
+  graphEditorMode: GraphEditorMode;
```

Comment:
the fact that we have `graphState: EditorGraphState` really burns me up, but I think for simplicity, we can leave them be for now, I guess when we do https://github.com/finos/legend-studio/issues/317 we can remove, `editorMode`, `mode` and `graphState`, for `EditorGraphState` after this change contains little state left, only serving as an organizer of methods